src/app/app.component.html

```diff
@@ -35,6 +35,12 @@
       <mat-divider></mat-divider>
     </mat-nav-list>
 
+    <app-popular-recipients
+        (recipientClicked)="searchFor($event)"
```

Comment:
Should this be "from:$event" ? Cos like this it finds mentions of the address inside other emails too